__tests__/app.test.js

```diff
@@ -89,6 +89,49 @@ describe("GET /api/articles/:article_id", () => {
     })
 })
 
+describe("GET /api/articles/:article_id/comments", () => {
+    test("200 - returns an array of comment objects of a specified article", () => {
+        return request(app)
+            .get("/api/articles/1/comments")
+            .expect(200)
+            .then(({ body }) => {
```

Comment:
There is an assertion that needs to be here 👀  we need an assertion to check that the array has a length property.